### PR TITLE
Feature/326 notify the user at the next step

### PR DESCRIPTION
**Pull request by dkrakau**
### Description

ISSUE: #326 notify the user at the next step

## Checklist

- [x] My code follows the style guidelines
- [ ] I have documented my code
- [x] My changes generate no new warnings
- [x] I have linked the Issue
- [ ] The Pull Request covers all the Checkpoints from the tagged Issue
- [x] I have noted the hours of work i have put in the issue

## I have edited following Parts of the project

- [x] Frontend
- [ ] Backend
- [ ] Github
- [ ] Root

## If tests are required did you make them?

- [ ] No tests required
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by AconexOfficial** [comment]
Is this pull request ready for review, since you already assigned reviewers?

File: frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    const currentPositionCoordinates: LocationCoordinates = {
+      latitude: currentLocation.location?.coords.latitude,
+      longitude: currentLocation.location?.coords.longitude
+    } as LocationCoordinates;
+
+    locations.map(location => {
+      if (isCurrentLocationArrivedAtStopLocation(currentPositionCoordinates, location.details.coordinates)) {
+        setDisplayArrivedNotification(true);
+        setArrivedNotificationMessage("Sie haben die Haltestelle " + location.name + " erreicht.");
+        console.log("Sie haben die Haltestelle " + location.name + " erreicht.");
```
Comment (AconexOfficial) [inline frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx:53]: Remove log

File: frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      }
+    });
+  };
+
+  useEffect(() => {
+    intervalId.current = setInterval(() => {
+      checkPosition();
+      console.log("checking position");
```
Comment (AconexOfficial) [inline frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx:61]: Remove log

File: frontend/src/components/Menu.tsx [hunk 3]
```diff
@@ -52,3 +52,12 @@ const Menu: React.FC<MenuProps> = (props) => (
...
 
 export default Menu;
+
+export const cleanAllIntervals = (): void => {
+  const intervalId = window.setInterval(function () { }, Number.MAX_SAFE_INTEGER);
+  // Clear any timeout/interval up to that id
+  for (let i = 1; i < intervalId; i++) {
+    window.clearInterval(i);
+  }
+  console.log("All intervals cleared.");
```
Comment (AconexOfficial) [inline frontend/src/components/Menu.tsx:62]: Remove log

File: frontend/src/pages/LiveNavigation/LiveNavigation.tsx [hunk 4]
```diff
@@ -11,44 +13,65 @@ import { blackListJourney, findJourneyFromNextStop, parseJSONToJourney } from ".
...
+          className="back-button"
+          onClick={() => {
+            window.localStorage.removeItem("selectedJourney");
+            cleanAllIntervals();
+            history.back();
+          }}
           size="default" expand="block">
           Navigation Beenden
         </IonButton>
+        {/* <IonButton onClick={() => setDisplayArrivedNotification(!displayArrivedNotification)}>show toast</IonButton> */}
```
Comment (AconexOfficial) [inline frontend/src/pages/LiveNavigation/LiveNavigation.tsx:59]: Remove comment

**Review by AconexOfficial — CHANGES_REQUESTED** [review]

File: frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx (hunk 1 above)
Comment (dkrakau) [reply frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx]: Log is removed.

File: frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx (hunk 2 above)
Comment (dkrakau) [reply frontend/src/components/LiveNavigationInfo/StopCheckerComponent.tsx]: Log is removed.

File: frontend/src/components/Menu.tsx (hunk 3 above)
Comment (dkrakau) [reply frontend/src/components/Menu.tsx]: Log is removed.

File: frontend/src/pages/LiveNavigation/LiveNavigation.tsx (hunk 4 above)
Comment (dkrakau) [reply frontend/src/pages/LiveNavigation/LiveNavigation.tsx:59]: Log is removed.

**Review by AconexOfficial — APPROVED** [review]

**Pull request merged**